[2024-03-04T15:39:32.591+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hola_mundo.branch_b manual__2024-03-04T15:39:05.640736+00:00 [queued]>
[2024-03-04T15:39:32.722+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hola_mundo.branch_b manual__2024-03-04T15:39:05.640736+00:00 [queued]>
[2024-03-04T15:39:32.723+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-03-04T15:39:32.939+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): branch_b> on 2024-03-04 15:39:05.640736+00:00
[2024-03-04T15:39:32.971+0000] {standard_task_runner.py:60} INFO - Started process 176 to run task
[2024-03-04T15:39:32.978+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hola_mundo', 'branch_b', 'manual__2024-03-04T15:39:05.640736+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/prueba.py', '--cfg-path', '/tmp/tmpqdhvvwnx']
[2024-03-04T15:39:33.006+0000] {standard_task_runner.py:88} INFO - Job 191: Subtask branch_b
[2024-03-04T15:39:33.362+0000] {task_command.py:423} INFO - Running <TaskInstance: hola_mundo.branch_b manual__2024-03-04T15:39:05.640736+00:00 [running]> on host d65f2c38c452
[2024-03-04T15:39:34.061+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='guille' AIRFLOW_CTX_DAG_ID='hola_mundo' AIRFLOW_CTX_TASK_ID='branch_b' AIRFLOW_CTX_EXECUTION_DATE='2024-03-04T15:39:05.640736+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-04T15:39:05.640736+00:00'
[2024-03-04T15:39:34.082+0000] {logging_mixin.py:188} INFO - Ejecutando la segunda rama
[2024-03-04T15:39:34.082+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-04T15:39:34.130+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hola_mundo, task_id=branch_b, execution_date=20240304T153905, start_date=20240304T153932, end_date=20240304T153934
[2024-03-04T15:39:34.350+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-04T15:39:34.595+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
